### PR TITLE
Update Table.md

### PR DESCRIPTION
**Pull request by M3NT8L-One**
Test

### CODE REVIEW

File: Table.md [hunk 1]
```diff
@@ -3,4 +3,6 @@ This is a table
 Field 1|Field 2|Field 3
 -------|-------|-------
 test|cat|dog
-Test2|lobster|shrimp
\ No newline at end of file
+Test2|lobster|shrimp
+
+!!!! GO !!!!
```
Comment (MirageC79) [inline Table.md:8]: Go Habs Go!

Comment (M3NT8L-One) [reply Table.md:8]: Pack sand

**Comment by MirageC79** [comment]
Was great forking with you!

**Comment by MirageC79** [comment]
At least I did not wreck the truck!

**Pull request merged**